packages/styles/package.json

```diff
@@ -36,7 +36,7 @@
     "@carbon/layout": "10.7.0",
     "@carbon/themes": "10.9.0",
     "@carbon/type": "10.5.1",
-    "carbon-components": "10.9.0"
+    "carbon-components": "10.9.3-rc.0"
```

Comment:
@annawen1 is there a date when they are doing the full `10.9.3` release? We probably should create a followup story to upgrade once it's out of rc.

Reply:
hmm based on the comments in the [bug ticket](https://github.com/carbon-design-system/carbon/issues/5073) doesn't seem like they have a date in mind yet

Reply:
Should we potentially hold off on upgrading this then? This update affects all components/patterns.

Reply:
ok I'm good with waiting for full release

Reply:
reopening this PR with the latest carbon dependency versions